docs/prover/cli/options.md

```diff
@@ -328,6 +328,50 @@ The default number of loop iterations we unroll is one. However, in many cases,
 certoraRun Bank.sol --verify Bank:Bank.spec --loop_iter 2
 ```
 
+Options regarding hashing of unbounded data
+-------------------------------------------
+
+(--optimistic_hashing)=
+### `--optimistic_hashing`
+
+**What does it do?**
+
+ When hashing data of potentially unbounded length (including unbounded arrays, like `bytes`, `uint[]`, etc.), assume that its length is bounded by the value set through the `--hashing_length_bound` option. If this is not set, and the length can be exceeded by the input program, the prover reports an assertion violation. I.e., when this option is set, the boundedness of the hashed data assumed checked by the prover, when this option is set that boundedness is assumed instead.
+
```

Comment:
I would include a pointer to the other document here. (e.g.
```
See {doc}`../approx/hashing` for more details.
```

Reply:
good point, done